Sources/MapboxDirections/DirectionsOptions.swift

```diff
@@ -218,6 +218,18 @@ open class DirectionsOptions: Codable {
         }
     }
     
+    /**
+     Initializes an options object for routes between the given waypoints and an optional profile identifier.
+
+     Do not call `DirectionsOptions(waypoints:profileIdentifier:)` directly; instead call the corresponding initializer of `RouteOptions` or `MatchOptions`.
```

Comment:
Since developers should not call `Directionsoptions(waypoints:profileIdentifier:)` directly, is it safe to not include this change?

Reply:
We should include it, because it is valid, if somewhat contrived, to subclass DirectionsOptions directly. The use case would be building support for another navigation API service endpoint. Normally that’s something this library would have built right in, but in principle, client code could implement that support all the same.